fluentd/configs.d/openshift/output-applications.conf

```diff
@@ -0,0 +1,5 @@
+<match **>
+   @type copy
+   @include output-es-config.conf
+   @include ../dynamic/es-copy-config.conf
```

Comment:
The only catch with a user wanting to configure fluentd to send to anything
extra is that we'd need to make sure our image provides that plugin...

On Tue, Jul 12, 2016 at 4:21 PM, Richard Megginson <notifications@github.com
> wrote:

> In fluentd/configs.d/openshift/output-applications.conf
> <https://github.com/openshift/origin-aggregated-logging/pull/195#discussion_r70524938>
> :
>
> > @@ -0,0 +1,5 @@
> > +<match **>
> > +   @type copy
> > +   @include output-es-config.conf
> > +   @include ../dynamic/es-copy-config.conf
>
> I suggest to have an @include output-extra-*.conf here - then the user
> could add a config map for output-extra-splunk.conf or
> output-extra-fluentd-secure-forward.conf or whatever
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/195/files/c79a4f8ff336fad4187dcb823b73638724df6f84#r70524938>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABXIEf0T3MbS75dk4DtWD-Cb9MZ_l0oNks5qVAVcgaJpZM4JHWV8>
> .
>



-- 
Eric Wolinetz

Red Hat Engineering | OpenShift
ewolinet@redhat.com
IRC: ewolinetz
C: (847) 641-0143